clang/include/clang/Driver/Options.td

```diff
@@ -5703,6 +5703,11 @@ def print_supported_extensions : Flag<["-", "--"], "print-supported-extensions">
   Visibility<[ClangOption, CC1Option, CLOption]>,
   HelpText<"Print supported -march extensions (RISC-V, AArch64 and ARM only)">,
   MarshallingInfoFlag<FrontendOpts<"PrintSupportedExtensions">>;
+def print_enabled_extensions : Flag<["-", "--"], "print-enabled-extensions">,
+  Visibility<[ClangOption, CC1Option, CLOption]>,
+  HelpText<"Print the -march/-mcpu extensions enabled for the given target"
```

Comment:
This is tricky to word without it becoming a paragraph, but perhaps something like this is more accurate:
```
Print the extensions enabled by the given target and -march/-mcpu options.
```
This distinguishes it from `--print-supported-extensions`. Phrasing aside, the point is that it's listing a consequence of -march/-mcpu, not what you could add to -march and -mcpu.

`supported` is "here's what you could have", `enabled` is "this is what we gave you given the other options you used".

Reply:
Done.